src/components/WalletCompare.js

```diff
@@ -187,6 +187,9 @@ const WalletCompare = ({ location }) => {
           has_high_volume_purchases
           has_dex_integrations
           has_multisig
+          image {
+            ...walletCardImage
```

Comment:
Wait what!? We can now use dynamic image references?? 🤩

Reply:
Yes! pretty cool. Would like to slowly transition to this on the rest of the queries.